tslint.json

```diff
@@ -1,97 +1,40 @@
-            "variables-before-functions"
-        ],
-        "no-any": false,
-        "no-arg": true,
-        "no-bitwise": true,
-        "no-console": [
-            true,
-            "log",
-            "error",
-            "debug",
-            "info",
-            "time",
-            "timeEnd",
-            "trace"
-        ],
-        "no-consecutive-blank-lines": true,
-        "no-construct": true,
-        "no-debugger": true,
-        "no-duplicate-variable": true,
-        "no-empty": true,
-        "no-eval": true,
-        "no-shadowed-variable": true,
-        "no-string-literal": true,
-        "no-switch-case-fall-through": true,
-        "no-trailing-whitespace": false,
-        "no-unused-expression": true,
-        "no-use-before-declare": true,
-        "one-line": [
-            true,
-            "check-catch",
-            "check-else",
-            "check-open-brace",
-            "check-whitespace"
-        ],
-        "quotemark": [true, "single", "jsx-double"],
-        "radix": true,
-        "semicolon": [true, "never"],
-        "switch-default": true,
-        "trailing-comma": [false],
-        "triple-equals": [ true, "allow-null-check" ],
-        "typedef": [
-            true,
-            "parameter",
-            "property-declaration"
-        ],
-        "typedef-whitespace": [
-            true,
-            {
-                "call-signature": "nospace",
-                "index-signature": "nospace",
-                "parameter": "nospace",
-                "property-declaration": "nospace",
-                "variable-declaration": "nospace"
-            }
-        ],
-        "variable-name": [true, "ban-keywords", "check-format", "allow-leading-underscore", "allow-pascal-case"],
-        "whitespace": [
-            true,
-            "check-branch",
-            "check-decl",
-            "check-module",
-            "check-operator",
-            "check-separator",
-            "check-type",
-            "check-typecast"
-        ]
-    }
+  "extends": [
+    "tslint:latest",
+    "tslint-react",
+    "tslint-config-prettier",
+    "tslint-plugin-prettier"
+  ],
+  "rules": {
+    "object-literal-sort-keys": false,
+    "ban": false,
+    "ban-types": false,
+    "ordered-imports": false,
+    "no-submodule-imports": false,
+    "no-any": false,
+    "member-access": false,
+    "object-literal-sort-keys": false,
+    "no-arg": true,
+    "interface-name": [
+      true,
+      "never-prefix"
+    ],
+    "variable-name": [
+      "allow-leading-underscore"
+    ],
+    "sort": false,
+    "no-bitwise": true,
+    "no-console": false,
+    "interface-over-type-literal": false,
+    "no-debugger": true,
+    "no-duplicate-variable": true,
+    "no-eval": true,
+    "curly": true,
+    "no-unused-expression": false,
+    "jsx-no-lambda": false,
+    "jsx-no-multiline-js": false,
+    "jsx-boolean-value": false,
+    "no-shadowed-variable": true,
+    "prettier": true
```

Comment:
Prettier now runs as a tslint rule, and all codestyle rules are turned off in tslint. Although the linting does seem to take a bit longer, it's more robust this way.